x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/Waterfall/waterfall_item.tsx

```diff
@@ -44,7 +45,10 @@ const Container = euiStyled.div<IContainerStyleProps>`
   padding-top: ${({ theme }) => theme.eui.paddingSizes.s};
   padding-bottom: ${({ theme }) => theme.eui.euiSizeM};
   margin-right: ${(props) => props.timelineMargins.right}px;
-  margin-left: ${(props) => props.timelineMargins.left}px;
+  margin-left: ${(props) =>
+    props.hasToggle
+      ? props.timelineMargins.left - 30 // fix margin if there is a toggle
```

Comment:
is this 30px value consistent with different zoom levels?

Reply:
30px is the width of the toggle, I would have love to test it with different levels, but I couldn't find data for cases with more traces.